charms/katib-db-manager/src/charm.py

```diff
@@ -4,12 +4,22 @@
 
 import logging
 
-from oci_image import OCIImageResource, OCIImageResourceError
+from charmed_kubeflow_chisme.exceptions import ErrorWithStatus, GenericCharmRuntimeError
+from charmed_kubeflow_chisme.kubernetes import KubernetesResourceHandler
+from charmed_kubeflow_chisme.lightkube.batch import delete_many
+from charmed_kubeflow_chisme.pebble import update_layer
+from charms.observability_libs.v1.kubernetes_service_patch import KubernetesServicePatch
+from lightkube import ApiError
+from lightkube.generic_resource import load_in_cluster_generic_resources
+from lightkube.models.core_v1 import ServicePort
 from ops.charm import CharmBase
 from ops.main import main
 from ops.model import ActiveStatus, MaintenanceStatus, WaitingStatus
+from ops.pebble import Layer
 
-logger = logging.getLogger(__name__)
+K8S_RESOURCE_FILES = [
+    "src/templates/auth_manifests.yaml.j2",
+]
 
 
 class CheckFailed(Exception):
```

Comment:
Is this used after re-write?

Reply:
yes, it's passed to the KubernetesResourceHandler

Reply:
Hmm, I could not find `CheckFailed` to be used anywhere in `charm.py`. Am I missing something?
I thought we changed all that error handling to use `ErrorWithStatus`.

Reply:
Got it. I was talking about `CheckFailed` class. Let's remove it.